lms/static/js/jquery.gradebook.js

```diff
@@ -27,49 +21,38 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		var targetLeft = clamp(tableOrigin + offset, maxScroll, 0);
 
 		updateHorizontalPosition(targetLeft);
-
-		setShadows(targetLeft);
 	};
 
-	var stopDrag = function(e) {
+	var stopDrag = function() {
 		$body.css('-webkit-user-select', 'auto');
 		$body.unbind('mousemove', moveDrag);
 		$body.unbind('mouseup', stopDrag);
 	};
 
-	var setShadows = function(left) {
-		var padding = 30;
-
-		var leftPercent = clamp(-left / padding, 0, 1);
-		$leftShadow.css('opacity', leftPercent);
-	
-		var rightPercent = clamp((maxScroll + left) / padding, 0, 1);
-		$rightShadow.css('opacity', rightPercent);
-	};
-
 	var clamp = function(val, min, max) {
```

Comment:
You may want to use `Math.max` and `Math.min` to define this function, [as suggested by this StackOverflow question](http://stackoverflow.com/questions/11409895/whats-the-most-elegant-way-to-cap-a-number-to-a-segment). It's clearer and more performant.